docs/json.md

```diff
@@ -154,6 +154,9 @@ If a field is to be declared as "required" in the JSON Schema, the annotation
 ```
 is used.
 
+Please note, we do not recommend using the `$required` flag for boolean fields, as not setting it is equivalent to `false`. 
```

Comment:
Usually, I think we call all the $...-stuff "annotation".

```suggestion
Please note, we do not recommend using the `$required` annotation for boolean fields, as not setting it is equivalent to `false`. 
```

Reply:
Yes, you are right. Changed it.